migrations/20240501024340_vnft_into_ud_tbl.sql

```diff
@@ -0,0 +1,109 @@
+-- +goose Up
+-- +goose StatementBegin
+SET search_path = devices_api, public;
+
+ALTER TABLE user_devices
+    ADD COLUMN mint_request_id char(27) CONSTRAINT user_devices_mint_request_id_fkey UNIQUE REFERENCES meta_transaction_requests(id);
```

Comment:
I didn't know you could combine and the foreign key. Could we separate these out? Give them both names (Postgres will give them names anyway, I just like having them written out).